cypress/support/step_definitions/dictionary/addConceptInDictionaryPage/addconceptInDictioanryPage.ts

```diff
@@ -0,0 +1,74 @@
+/// <reference types="cypress" />
+/// <reference types="../../../" />
+import {
+  Given,
+  Then,
+  When
+} from "cypress-cucumber-preprocessor/steps";
+import { getDictionaryId, getUser, getConceptId } from "../../../utils";
+
+
+Given("the user is on the view dictionary concepts page", () => {
+  cy.visit(`/users/${getUser()}/collections/${getDictionaryId()}/concepts/`);
+});
+
+When('the user clicks the "Add concepts" button', () => {
+  cy.findByTitle("Add concepts").click();
+});
+
+Then('the user selects "Import existing concept"', () => {
+  cy.visit(`/orgs/CIEL/sources/CIEL/concepts/?addToDictionary=/users/${getUser()}/collections/${getDictionaryId()}/`);
+});
+
+Then('the user should be on the "Import existing concept" page', () => {
+  cy.url().should("contain", "/orgs/CIEL/sources/CIEL/concepts/");
+});
+
+Given('the user is on the "Import existing concept" page', () => {
+  cy.visit(`/orgs/CIEL/sources/CIEL/concepts/?addToDictionary=/users/${getUser()}/collections/${getDictionaryId()}/`);
+});
+
+When('the user clicks on the row for "Serum"', () => {
+  cy.findByText("Serum").click();
```

Comment:
cypress throws the following error, 

```
Unable to find an element with the text: Serum. This could be because the text is broken up by multiple elements. In this case, you can provide a function for your text matcher to make your matcher more flexible.
```
as described in the error message, it seems there's no such element with the text "Serum"

Reply:
Yes, to address this, we actually need to create a concept called "Serum" in a source called "CIEL" owned by an organisation called "CIEL". These tests are run against an empty instance of OCL (i.e., OCL without any content).

This will probably require a new command (since the `createSource` command only supports created a user source).

So we need something called before this function that does something like this:

```ts
cy.createOrganisation("CIEL", true).createOrgSource("CIEL", "CIEL", true);
```

And then afterwards,

```ts
cy.deleteOrganisation("CIEL", true);
```

Will also need a `createConcept` command that takes the source that the concept belongs to and the minimum number of properties we need so we can create a "Serum" concept before this test. Does that make some kind of sense?